docs/gettsim_objects/input_variables.rst

```diff
@@ -101,3 +101,29 @@ household.
 +-------------------------+---------------------------------------------+--------------+
 | _`priv_rente_m`         | Amount of monthly private pension           | float        |
 +-------------------------+---------------------------------------------+--------------+
+| _`weiblich`             | True if female                              | bool         |
++-------------------------+---------------------------------------------+--------------+
+| _`rententrechtl_zeit`   | Sum of all times relevant for retirement    | float        |
```

Comment:
Are all these variables on a monthly level? Then we should append `_m` and mention it in the description. If not we might want to change it to months for all to be consistent.

Reply:
I have been considering these as fixed variables (as they are in the pension data - at the date of pension entry).